ansible/roles/session-valkey/tasks/main.yml

```
@@ -0,0 +1,68 @@
+- name: Secret for schulcloud valkey
+  kubernetes.core.k8s:
+    kubeconfig: ~/.kube/config
+    namespace: '{{ NAMESPACE }}'
+    template: onepassword.yml.j2
+  when: ONEPASSWORD_OPERATOR is defined and ONEPASSWORD_OPERATOR|bool
+  tags:
+    - 1password
+
+- name: Install valkey sentinel
+  kubernetes.core.helm:
+    chart_ref: oci://docker.io/bitnamicharts/valkey
+    chart_version: '{{ SESSION_VALKEY_CHART_VERSION }}'
+    release_name: session-valkey
+    release_namespace: '{{ NAMESPACE }}'
+    release_state: present
+    create_namespace: yes
+    kubeconfig: ~/.kube/config
+    update_repo_cache: no
+    values:
+      global:
+        defaultStorageClass: '{{ SC_DEFAULT_STORAGE_CLASS_NAME }}'
+      replica:
+        replicaCount: '{{ SESSION_VALKEY_REPLICAS }}'
+        pdb:
+          create: false
+        resources:
+          limits:
+            cpu: "{{ SESSION_VALKEY_CPU_LIMITS|default('1000m', true) }}"
+            memory: "{{ SESSION_VALKEY_MEMORY_LIMITS|default('4Gi', true) }}"
+          requests:
+            cpu: "{{ SESSION_VALKEY_CPU_REQUESTS|default('100m', true) }}"
+            memory: "{{ SESSION_VALKEY_MEMORY_REQUESTS|default('1Gi', true) }}"
+      primary:
+        pdb:
+          create: false
+        readinessProbe:
+          timeoutSeconds: 5
+        resources:
+          limits:
+            cpu: "{{ SESSION_VALKEY_CPU_LIMITS|default('1000m', true) }}"
+            memory: "{{ SESSION_VALKEY_MEMORY_LIMITS|default('4Gi', true) }}"
+          requests:
+            cpu: "{{ SESSION_VALKEY_CPU_REQUESTS|default('100m', true) }}"
+            memory: "{{ SESSION_VALKEY_MEMORY_REQUESTS|default('1Gi', true) }}"
+      auth:
+        existingSecret: session-valkey-password
+        existingSecretPasswordKey: SESSION_VALKEY_SENTINEL_PASSWORD
+        usePasswordFiles: false
+      sentinel:
+        enabled: true
+        readinessProbe:
+          timeoutSeconds: 5
+      metrics:
+        enabled: true
+        podMonitor:
+          enabled: true
+        serviceMonitor:
+          enabled: true
+      # OPS-6762 still up to debate if we will enable this
```

Adding of jwt count for active users count
```suggestion
      metrics:
        enabled: true
        podMonitor:
          enabled: true
        serviceMonitor:
          enabled: true
        extraArgs:
          check-key-groups: "(jwt):.+"
      # OPS-6762 still up to debate if we will enable this
```